app/controllers/reports_controller.rb

```diff
@@ -49,6 +49,25 @@ def performance_profiles
     end
   end
 
+  def bulk_qts_export
```

Comment:
i wonder whether all these methods and class names should be `bulk_recommend_export`, `bulk_recommend_trainees` etc since that's the action we're performing (we recommend them, dqt awards them). Also they can be recommended for QTS or EYTS, so feels nicer to not mention either of the statuses. whaddya think?

Reply:
yeah agreed, that sounds more accurate